usaspending_api/api_contracts/contracts/agency/AgencyProfile.md

```diff
@@ -164,7 +168,8 @@ This endpoint returns the amount that the specific agency has obligated to vario
 + obligated_amount: 209728763.65 (required, string)
 
 ## FederalAccount (object)
-+ account_title: `Refunding Internal Revenue Collections (Indefinite), Treasury` (required, string)
-+ id: 1105 (required, string)
++ account_title (required, string)
++ account_number (required, string)
++ id (required, string)
     The USAspending.gov unique identifier for the federal account. You will need to use this ID when making API requests for details about specific federal accounts.
-+ obligated_amount: 185853348344.60 (required, string)
++ obligated_amount (required, string)
```

Comment:
It would probably make more sense to change this to a number at some point.

Reply:
That makes sense to me as well (unless the formatting for that is done on the backend), but if it's currently ingested as a string (correct me if I'm wrong) it would be better to keep the contract only promising behavior that it actually provides. I added https://federal-spending-transparency.atlassian.net/browse/DEV-2577 as a skeleton ticket for later grooming.